docs/cli-reference/standalone-config.md

```diff
@@ -0,0 +1,43 @@
+# `spacetimedb-standalone` configuration
+
+A local database instance (as started by `spacetime start`) can be configured in `{data-dir}/config.toml`, where `{data-dir}` is the database's data directory. This directory is printed when you run `spacetime start`:
+
+<pre>spacetimedb-standalone version: 1.0.0
+spacetimedb-standalone path: /home/user/.local/share/spacetime/bin/1.0.0/spacetimedb-standalone
+database running in data directory <b>/home/user/.local/share/spacetime/data</b></pre>
```

Comment:
Have you tested that the website renders `pre` tags in a sensible way?

Reply:
No, but I'm not sure how to.